client/auth/login.jsx

```diff
@@ -22,47 +22,82 @@ import Main from 'calypso/components/main';
 import Notice from 'calypso/components/notice';
 import SelfHostedInstructions from './self-hosted-instructions';
 import WordPressLogo from 'calypso/components/wordpress-logo';
-import { login } from 'calypso/lib/oauth-store/actions';
 import { recordGoogleEvent } from 'calypso/state/analytics/actions';
 import { localizeUrl } from 'calypso/lib/i18n-utils';
+import * as OAuthToken from 'calypso/lib/oauth-token';
+import { errors as errorTypes, makeAuthRequest, bumpStats } from './login-request';
```

Comment:
You could rename the export to `errorTypes` at the source if that's how you prefer to import it later. Avoids the alias.